git-pluck

```diff
@@ -62,11 +62,20 @@ def main():
             highlight(bcolors.BLUE, 'Entering'),
             highlight(bcolors.ORANGE, os.path.relpath(root))
         )
+        current_repo = os.path.relpath(root)
```

Comment:
If you set this variable a little higher up it can be used in place of the other call to `os.path.relpath()` on line 63.